src/zenml/logging/step_logging.py

```diff
@@ -584,3 +587,60 @@ def wrapped_flush(*args: Any, **kwargs: Any) -> Any:
             return output
 
         return wrapped_flush
+
+
+def setup_orchestrator_logging(
+    run_id: str,
+    title: str = "orchestrator",
+) -> Any:
+    """Set up logging for an orchestrator environment.
+
+    This function can be reused by different orchestrators to set up
+    consistent logging behavior.
+
+    Args:
+        run_id: The pipeline run ID.
+        title: Title for the orchestrator logs.
+
+    Returns:
+        The logs context (PipelineLogsStorageContext)
+    """
+    try:
+        # Fetch the active stack
+        client = Client()
+        active_stack = client.active_stack
+
+        # Configure the logs
+        logs_uri = prepare_logs_uri(
+            artifact_store=active_stack.artifact_store,
+        )
+
+        logs_context = PipelineLogsStorageContext(
```

Comment:
This is missing a check whether logging is disabled via the env variable or the setting, no?